actionmailer/test/test_helper_test.rb

```diff
@@ -393,6 +399,37 @@ def test_assert_enqueued_email_with_with_parameterized_args
     end
   end
 
+  def test_assert_enqueued_email_with_parameterized_mailer
+    assert_nothing_raised do
+      assert_enqueued_email_with TestHelperMailer.with(all: "good"), :test_parameter_args do
+        silence_stream($stdout) do
+          TestHelperMailer.with(all: "good").test_parameter_args.deliver_later
+        end
+      end
+    end
+  end
+
+  def test_assert_enqueued_email_with_named_args
+    assert_nothing_raised do
+      assert_enqueued_email_with TestHelperMailer, :test_named_args, args: [{ email: "some_email", name: "some_name" }] do
+        silence_stream($stdout) do
+          TestHelperMailer.test_named_args(email: "some_email", name: "some_name").deliver_later
+        end
+      end
+    end
+  end
+
+
+  def test_assert_enqueued_email_with_params_and_args
```

Comment:
Do you think it needs another test without args and only params? `test_asset_enqueued_email_with_params` It might be excessive, but figured I'd ask.

Reply:
Yeah! Maybe I deleted the old one. You wanna add it?

Reply:
@brydave different than lines 392 and 402?

Reply:
@ron-shinall *maybe*? I was thinking lines 392 and 402 pass `args`, but we could add a test that explicitly passes `params` without `args`. Just a thought! I can create a branch and PR it separately (like @tomrossi7 mentioned in the software writer campfire) for review.